hanfor/requirements/requirements.py

```diff
@@ -86,9 +85,15 @@ def api_update():
     # Update a requirement
     rid = request.form.get("id", "")
     requirement = current_app.db.get_object(Requirement, rid)
+    order = request.form.get("formalizations_order") or "{}"
+    order_dict = json.loads(order)
+    logging.debug(order_dict)
+    for idx, formalization in requirement.formalizations.items():
+        formalization.order = order_dict.get(str(idx)) or 0
+        logging.debug(f"Formalizaation of {idx} has order of {formalization.order}")
+
```

Comment:
Do we need loggin for the order? Does not seem like a particularly interesting thing here.
Also do we expect no order to be available regularly and would this not be just a bug in the gui? So whys do we silently assume defaults here instead of, at least, logging that something is errorneous?